boards/nucleo32-l031/include/periph_conf.h

```diff
@@ -153,7 +153,15 @@ static const spi_conf_t spi_config[] = {
  * @name    ADC configuration
  * @{
  */
-#define ADC_NUMOF           (0)
+#define ADC_CONFIG {             \
+    { GPIO_PIN(PORT_A, 0), 0 },  \
```

Comment:
Please put comments about the on-board PIN label (`A0`, `A1`, etc). `A7` and `A5` are missing in the configuration btw.